serverless_llm/store/README.md

```diff
@@ -1 +1,37 @@
-# serverless_llm-store
\ No newline at end of file
+# serverless_llm-store
+
+## About
+
+`sllm-store` is an internal library of ServerlessLLM that provides high-performance model loading from local storage into GPU memory. You can also install and use this library in your own projects, following our [quick start guide](https://serverlessllm.github.io/docs/stable/store/quickstart).
```

Comment:
Could you also please update the docs?

Reply:
I mean add an "install from source" subsection.

Reply:
Updated that under "install with pip"